pkg/deploy/registry/deployconfig/strategy.go

```diff
@@ -76,7 +136,7 @@ func Matcher(label labels.Selector, field fields.Selector) generic.Matcher {
 		GetAttrs: func(obj runtime.Object) (labels.Set, fields.Set, error) {
 			deploymentConfig, ok := obj.(*api.DeploymentConfig)
 			if !ok {
-				return nil, nil, fmt.Errorf("not a DeploymentConfig")
+				return nil, nil, fmt.Errorf("not a deployment config")
```

Comment:
If we expect these to be user-facing errors, I think "configuration" over "config" would be good.